metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/variable_save_test.clj

```diff
@@ -0,0 +1,41 @@
+(ns cmr.metadata-db.int-test.concepts.variable-save-test
+  "Contains integration tests for saving variables. Tests saves with various configurations including
+  checking for proper error handling."
+  (:require [clojure.test :refer :all]
+            [clj-http.client :as client]
```

Comment:
For all the files I've been touching lately, I've been updating the `require`s to use the new indent style we've adopted; probably want to do that here, too.